# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv-cross-internal/src/vendor/SPIRV-Cross/Makefile
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv-cross-internal/src/vendor/SPIRV-Cross/Makefile b/third_party/rust/spirv-cross-internal/src/vendor/SPIRV-Cross/Makefile
deleted file mode 100644
--- a/third_party/rust/spirv-cross-internal/src/vendor/SPIRV-Cross/Makefile
+++ /dev/null
@@ -1,41 +0,0 @@
-TARGET := spirv-cross
-
-SOURCES := $(wildcard spirv_*.cpp)
-CLI_SOURCES := main.cpp
-
-OBJECTS := $(SOURCES:.cpp=.o)
-CLI_OBJECTS := $(CLI_SOURCES:.cpp=.o)
-
-STATIC_LIB := lib$(TARGET).a
-
-DEPS := $(OBJECTS:.o=.d) $(CLI_OBJECTS:.o=.d)
-
-CXXFLAGS += -std=c++11 -Wall -Wextra -Wshadow -Wno-deprecated-declarations
-
-ifeq ($(DEBUG), 1)
-	CXXFLAGS += -O0 -g
-else
-	CXXFLAGS += -O2 -DNDEBUG
-endif
-
-ifeq ($(SPIRV_CROSS_EXCEPTIONS_TO_ASSERTIONS), 1)
-	CXXFLAGS += -DSPIRV_CROSS_EXCEPTIONS_TO_ASSERTIONS -fno-exceptions
-endif
-
-all: $(TARGET)
-
--include $(DEPS)
-
-$(TARGET): $(CLI_OBJECTS) $(STATIC_LIB)
-	$(CXX) -o $@ $(CLI_OBJECTS) $(STATIC_LIB) $(LDFLAGS)
-
-$(STATIC_LIB): $(OBJECTS)
-	$(AR) rcs $@ $(OBJECTS)
-
-%.o: %.cpp
-	$(CXX) -c -o $@ $< $(CXXFLAGS) -MMD
-
-clean:
-	rm -f $(TARGET) $(OBJECTS) $(CLI_OBJECTS) $(STATIC_LIB) $(DEPS)
-
-.PHONY: clean